diff/__init__.py

```diff
@@ -0,0 +1,22 @@
+#* This file is part of MOOSETOOLS repository
+#* https://www.github.com/idaholab/moosetools
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+import logging
+
+from .diff import compare_jsons, validate_json_file, validate_tolerance
+from .moosedeepdiff import MOOSEDeepDiff
+
+# Configure logging to overwrite the log file for each run, add option: filemode='w'
+logging.basicConfig(filename='MOOSETools.log',
+                    level=logging.INFO,
+                    format='%(asctime)s %(levelname)s %(message)s',
+                    filemode='w',
+                    datefmt='%m/%d/%Y %H:%M:%S')
+
+logging.info('Application started. Logging to file MOOSETools.log')
```

Comment:
Don't do the configure here. That will get in the way of other tools who use this but want a different configuration. You can add this to the command line tool itself.

In each function use: `log = logging.getLogger(__name__)`, then log.error(), etc. This allows tools building from this to control the logging by function name, if desired.